docs/guide/robot_sbc/setup_jetson_nano.md

```diff
@@ -66,76 +136,364 @@ pip3 install -U gdown
 pip3 install --pre --extra-index-url https://developer.download.nvidia.com/compute/redist/jp/v45 tensorflow==2.3.1
 ```
 
-Finally, you can install PyTorch:
+#### Install Donkeycar Python Code
+
+Change to a dir you would like to use as the head of your projects. Assuming
+you've already made the `projects` directory above, you can use that. Get
+the latest 4.4.X release and install that into the venv.
 
 ```bash
-wget https://nvidia.box.com/shared/static/p57jwntv436lfrd78inwl7iml6p13fzh.whl
-cp p57jwntv436lfrd78inwl7iml6p13fzh.whl torch-1.8.0-cp36-cp36m-linux_aarch64.whl
-pip3 install torch-1.8.0-cp36-cp36m-linux_aarch64.whl
-sudo apt-get install libjpeg-dev zlib1g-dev libpython3-dev libavcodec-dev libavformat-dev libswscale-dev
-mkdir -p ~/projects; cd ~/projects
-git clone -b v0.9.0 https://github.com/pytorch/vision torchvision
-cd torchvision 
-python setup.py install
-cd ../
+mkdir projects
+cd ~/projects
+git clone https://github.com/autorope/donkeycar
+cd donkeycar
+git fetch --all --tags -f
+latestTag=$(git describe --tags `git rev-list --tags --max-count=1`)
+git checkout $latestTag
+pip install -e .[nano45]
+
+```
+
+### Step 5a: (Optional) Install PyGame for USB camera
+
+If you plan to use a USB camera, you will also want to setup pygame:
+
+```bash
+sudo apt-get install python-dev libsdl1.2-dev libsdl-image1.2-dev libsdl-mixer1.2-dev libsdl-ttf2.0-dev libsdl1.2-dev libsmpeg-dev python-numpy subversion libportmidi-dev ffmpeg libswscale-dev libavformat-dev libavcodec-dev libfreetype6-dev
+pip install pygame
 ```
 
-Optionally, you can install the RPi.GPIO clone for Jetson Nano from [here](https://github.com/NVIDIA/jetson-gpio). This is not required for default setup, but can be useful if using LED or other GPIO driven devices.
+Later on you can add the `CAMERA_TYPE="WEBCAM"` in myconfig.py.
 
 
 
-##  Step 5: Install Donkeycar Python Code
+## Installation for Donkey Car main
 
-* Change to a dir you would like to use as the head of your projects. Assuming you've already made the `projects` directory above, you can use that:
+Instructions for the latest code from the `main` branch. Note the 
+installation differs between the two available OSs. On Jetson Nano you will 
+need to install Jetpack 4.6.2. If you have newer Jetson board or a Jetson 
```

Comment:
The instructions are confusing around the Jetpack version.

> On Jetson Nano you will need to install Jetpack 4.6.2

is followed below by

> These instructions work for Jetpack 4.6.1. 

Nvidia do not have an SD download for 4.6.2:

> JetPack 4.6.2 can be installed using SDK Manager.

https://developer.nvidia.com/embedded/jetpack-sdk-462

I did find their instructions for updating an existing installation at https://docs.nvidia.com/jetson/archives/r34.1/DeveloperGuide/text/SD/SoftwarePackagesAndTheUpdateMechanism.html#updating-a-jetson-device

`sudo apt update; sudo apt upgrade`

Before:
```
$ head -n 1 /etc/nv_tegra_release
> # R32 (release), REVISION: 7.1, GCID: 29818004, BOARD: t210ref, EABI: aarch64, DATE: Sat Feb 19 17:05:08 UTC 2022
```
After:
```
$ head -n 1 /etc/nv_tegra_release
> R32 (release), REVISION: 7.3, GCID: 31982016, BOARD: t210ref, EABI: aarch64, DATE: Tue Nov 22 17:30:08 UTC 2022
```
where:

L4T 32.7.1 corresponds with JetPack 4.6.1
L4T 32.7.2 corresponds with JetPack 4.6.2

So I seem to be on a version past JetPack 4.6.2? I'm not entirely sure.

The documentation should add something along the lines

> upgrade from the 4.6.1 SD image using `sudo apt update; sudo apt upgrade`

Reply:
Good point, I have corrected the versions and added your comment about update/upgrade.